aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js

```diff
@@ -92,87 +89,58 @@ const filtersConfig = [
       valueRange: '0 - 40 hours',
     },
   },
-  { property: 'name', label: 'Name', filterType: 'CheckBoxGroup' },
 ];
 
-export const TableCustomizationExample = () => {
-  const [visibleColumns, setVisibleColumns] = useState(COLUMNS);
-  const [open, setOpen] = useState(false);
-
-  return (
-    <Page background="background" fill>
-      <PageContent>
-        <FiltersProvider>
-          <Box gap="medium">
-            <Header pad={{ top: 'medium' }}>
-              <Box gap="xsmall" fill="horizontal">
-                <Heading id="users-heading" level={2} margin="none">
-                  Users
-                </Heading>
-                <Box direction="row" justify="between" align="start" wrap>
-                  <FilterControls
-                    // Table column configuration should be grouped on right
-                    // side with other actions to separate it from filtering.
-                    // If grouped to the left with the Filter control, it
-                    // becomes confusing what the "Clear filters" button will
-                    // do with regards to any column configurations that have
-                    // been applied.
-                    configure={
-                      <DropButton
-                        a11yTitle="Configure columns button"
-                        icon={<Splits />}
-                        kind="toolbar"
-                        dropAlign={{ top: 'bottom', right: 'right' }}
-                        onClose={() => setOpen(false)}
-                        onOpen={() => setOpen(true)}
-                        dropContent={
-                          <ColumnSettings
-                            columns={COLUMNS}
-                            visibleColumns={visibleColumns}
-                            setVisibleColumns={setVisibleColumns}
-                            open={open}
-                          />
-                        }
-                        tip="Configure columns"
-                      />
-                    }
-                    actions={<Menu kind="toolbar" label="Actions" items={[]} />}
-                    data={allData}
-                    filters={filtersConfig}
-                    searchFilter={{ placeholder: 'Search' }}
-                  />
-                </Box>
-              </Box>
-            </Header>
-            <Results columns={visibleColumns} />
+export const TableCustomizationExample = () => (
+  <Page background="background" fill>
+    <PageContent>
+      <Box gap="medium">
+        <Header pad={{ top: 'medium' }}>
+          <Box gap="xsmall" fill="horizontal">
```

Comment:
Why is this `Box` needed here? There is only one child, so I don't see where `gap` would have any effect.